agent/resources/resources_linux.go

```diff
@@ -93,6 +109,14 @@ func (c *cgroupWrapper) setupCgroup(task *api.Task) error {
 		return errors.Wrapf(err, "resource: setup cgroup: unable to create cgroup at %s for task: %s", cgroupRoot, task.Arn)
 	}
 
+	// echo 1 > memory.use_hierarchy
+	memorySubsystem := c.memorySubsystemPath
+
+	err = c.ioutil.WriteFile(filepath.Join(memorySubsystem, cgroupRoot, memoryUseHierarchy), []byte(strconv.Itoa(1)), os.FileMode(0))
```

Comment:
Can you make the `1` and `0` constant and add some comment?

Reply:
Unless you feel strongly about it, I don't think this needs to have its own constants since they're only being used in this specific instance, also I think the comment ` echo 1 > ...` conveys context. Let me know though.

Reply:
should we do this for each task? Isn't it enough if we did this once at the top level `ecs` cgroup?

Reply:
@aaithal  you're right, ill move this up a level and make it part of `cgroupInit()`